src/Dockerfile

```diff
@@ -34,7 +34,7 @@ LABEL summary=$SUMMARY \
 {%- else %}
       version="{{ spec.version }}" \
 {%- endif %}
-      usage="docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 {{ spec.org }}/postgresql-{{ spec.short }}-{{ spec.prod }}" \
+      usage="docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 centos/postgresql-96-centos7" \
```

Comment:
Why this change?

Reply:
This was a mistake I created while rebasing, it should be OK now.